apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.ts

```diff
@@ -1958,6 +1957,10 @@ export class ProformaInvoiceComponent implements OnInit, OnDestroy, AfterViewIni
 
         this.invFormData.voucherDetails.tcsTotal = tcsSum;
         this.invFormData.voucherDetails.tdsTotal = tdsSum;
+        /** Call calculateBalanceDue if advance receipt adjusted for balance due added TDS TCS taxes  */
+        if (this.adjustPaymentData.totalAdjustedAmount) {
```

Comment:
calculateBalanceDue will not be called if totalAdjustedAmount is 0

Reply:
yes we no need to call in this condition